.github/workflows/goreleaser.yml

```diff
@@ -100,5 +98,4 @@ jobs:
     with:
       base64-subjects: "${{ needs.goreleaser.outputs.hashes }}"
       upload-assets: true
-      private-repository: true
-
+      private-repository: true
```

Comment:
Do we need this? vet is a public repository. What's the benefit of keeping this private?